src/verify.v2.integration.test.ts

```diff
@@ -47,8 +48,25 @@ const verifyRopsten = verificationBuilder(openAttestationVerifiers, { network: "
 const verifyRinkeby = verificationBuilder(openAttestationVerifiers, { network: "rinkeby" });
 
 describe("verify(integration)", () => {
+  beforeEach(() => {
+    jest.resetModules();
+    process.env = Object.assign(process.env, {
```

Comment:
store and restore the values just incase any other test is doing anything with these variables

Reply:
ok will do